README.md

```diff
@@ -19,7 +19,10 @@ This is a little description about your project.
 1. Go into the server directory `cd server`
 2. Run `npm install` to install packages
 3. Create your environment variable (.env) file
-4. Run `npm run dev` to start the server
+4. Go into the seeds directory `cd seeds`
+5. Run `node demoSeed.js` to create the demo user account
```

Comment:
Just came to my mind! Could we do a script in `package.json` to do `npm run seed`?

Reply:
Done in the latest version. 
- see update in package.json
- Updated README (just noticed numbering was off in the last version - sorry about that!)